pyfar/dsp/dsp.py

```diff
@@ -223,3 +224,75 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def deconv(measurement, excitation, **kwargs):
+    """Function to calculate the deconvolution of two signals.
+
+    Parameters
+    ----------
+    measurement : pyfar.Signal
+        The measurement signal, recorded after passing the device under test.
+    excitation : pyfar.Signal
+        The excitation signal, used to perform the measurement.
+    kwargs : Parameters for regu_inversion, can include:
+        freq_range, regu_outside, regu_inside, regu_final
+        refer to function regularized_spectrum_inversion for deatiled
+        documentation.
+
+    Returns
+    -------
+    pyfar.Signal
```

Comment:
This will reference the signal class in the Sphinx documentation :)
```suggestion
    signal : Signal
```
You should as well replace `pyfar.Signal` with `Signal` above...

Reply:
check

Reply:
check